### PR TITLE
Merge 'statistics' and 'GitHub Statistic

### LINKED ISSUE #3557: Merge 'statistics' and 'GitHub Statistics'
Right now, we have two sections in the sidebar of a package that deal with statistics:

screenshot from 2018-04-05 07-21-13

It would be nice to either group these together, or merge them into one section.

---

**Good First Issue**: This issue is good for first time contributors. If you've already contributed to Warehouse, please work on [another issue without this label](https://github.com/pypa/warehouse/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+-label%3A%22good+first+issue%22) instead. If there is not a corresponding pull request for this issue, it is up for grabs. For directions for getting set up, see our [Getting Started Guide](https://warehouse.pypa.io/development/getting-started/). If you are working on this issue and have questions, please feel free to ask them here, [`#pypa-dev` on Freenode](https://webchat.freenode.net/?channels=%23pypa-dev), or the [pypa-dev mailing list](https://groups.google.com/forum/#!forum/pypa-dev).

**Screenshot Required**: *If your pull request makes a visual change*, please include a screenshot of your update. This helps our team give you feedback faster.

### LINKED ISSUE #3754: add explanation of file download link structure to docs
We should add Ernest's explanation https://github.com/pypa/warehouse/issues/3738#issuecomment-382083526 to https://warehouse.readthedocs.io/api-reference/integration-guide/#integration-guide . The file to update is `docs/api-reference/integration-guide.rst`.

---

**Good First Issue**: This issue is good for first time contributors. If you've already contributed to Warehouse, please work on [another issue without this label](https://github.com/pypa/warehouse/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+-label%3A%22good+first+issue%22) instead. If there is not a corresponding pull request for this issue, it is up for grabs. For directions for getting set up, see our [Getting Started Guide](https://warehouse.pypa.io/development/getting-started/). If you are working on this issue and have questions, please feel free to ask them here, [`#pypa-dev` on Freenode](https://webchat.freenode.net/?channels=%23pypa-dev), or the [pypa-dev mailing list](https://groups.google.com/forum/#!forum/pypa-dev).

### PR DESCRIPTION
**Pull request by tomfbiz**
This fixes #3557  

Removed the second **Statistics** left nav section, and moved the libraries.io and Google BigQuery links under **Github Statistics**

screen shot 2018-05-14 at 11 11 54 am

### CODE REVIEW

**Comment by malikoth** [comment]
LGTM

File: warehouse/templates/packaging/detail.html [hunk 1]
```diff
@@ -164,6 +164,7 @@ <h3 class="sidebar-section__title">GitHub statistics</h3>
             <strong>Open issues/PRs: </strong>
             <span class="github-repo-info__item" data-key="open_issues_count"></span>
           </a>
+          <p>View additional statistics for this project via <a href="https://libraries.io/pypi/{{ release.project.name }}">Libraries.io</a>, or by using <a href="https://packaging.python.org/guides/analyzing-pypi-package-downloads/">Google BigQuery</a></p>
```
Comment (di) [inline warehouse/templates/packaging/detail.html]: This line will not be shown if the release doesn't have a `github_repo_info_url`. We'll need to add a fallback to the conditional which still renders the original section.

**Comment by tomfbiz** [comment]
Addressed @di comment. Non GitHub content shows up both for users with no gitub url like http://localhost/project/pygments-lexer-solidity/

and if github request fail (such as when the request fails like
http://localhost/project/jarmanifest/

image

**Comment by meahow** [comment]
@tomfbiz I believe you have misspelled the issue number. Should be #3557 and not #3754 ?

**Comment by di** [comment]
Temporarily canceling the Travis build here so we can get some higher priority things merged, I'll restart it after I review.

**Comment by di** [comment]
Thanks @tomfbiz!

**Pull request merged**